lib/fog/compute/kubevirt.rb

```diff
@@ -115,6 +119,18 @@ def deep_merge!(source_hash, other_hash, &block)
 
           source_hash
         end
+
+        # Updates a given VM raw entity with vm instance info if exists
+        #
+        # @param vm [Hash] A hash with vm raw data.
+        def populate_server_with_runtime_info(vm)
```

Comment:
This seems like something we could have in vm entity. Based on responsibilities seems like it would be better place there.

Reply:
I'm not sure if VM is the right place for it since it deals with setting values from VitrulaMachineInstance.
I can move this one to Server if you feel it is more right .

Reply:
Sounds great